DemoApp/Sources/Views/ReadableContentGuide/ReadableContentGuide.swift

```diff
@@ -0,0 +1,49 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+struct ReadableContentGuideViewModifier: ViewModifier {
```

Comment:
What's this used for?

Reply:
nice, but the second one on the right feels wrong though

Reply:
Yeah, the background color of the list is bugging me. I'm looking into it